{{cookiecutter.project_slug}}/src/{{cookiecutter.project_module}}/settings.py

```diff
@@ -105,4 +106,5 @@ def __init__(self):
         super().__init__()
         self.DEBUG = False
         self.SECRET_KEY = os.environ['SECRET_KEY']
+        self.SCRIPT_NAME = os.environ['SCRIPT_NAME']
```

Comment:
`SCRIPT_NAME` is part of wsgi `environ`, it doesn't need to be set in the settings